doc/releases/release-notes-2.3.rst

```diff
@@ -91,6 +91,13 @@ Stable API changes in this release
   * The net_idx parameter has been removed from the Health Client model
     APIs since it can be derived (by the stack) from the app_idx parameter
 
+* Networking
+
+  * The NET_DEVICE_INIT(), NET_DEVICE_INIT_INSTANCE() and ETH_NET_DEVICE_INIT()
```

Comment:
I think at least NET_DEVICE_OFFLOAD_INIT is missing from this list. Any others?

Reply:
> I think at least NET_DEVICE_OFFLOAD_INIT is missing from this list. Any others?

There was only 4 macros changed, I will add the missing one.